src/Compilers/Server/VBCSCompilerTests/VBCSCompilerServerTests.cs

```diff
@@ -370,7 +386,7 @@ public async Task AnalyzerInconsistencyShouldShutdown()
                     return new AnalyzerInconsistencyBuildResponse();
                 });
 
-                using var serverData = await ServerUtil.CreateServer(compilerServerHost: compilerServerHost).ConfigureAwait(false);
+                using var serverData = await ServerUtil.CreateServer(EmptyCompilerServerLogger.Instance, compilerServerHost: compilerServerHost).ConfigureAwait(false);
```

Comment:
Was it intentional to not pass `Logger` to this test?

Reply:
Nope that was an omission. Fixed.